addons/binding/org.openhab.binding.yeelight/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: yeelight Binding
+Bundle-SymbolicName: org.openhab.binding.yeelight;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/yeelight_lib.jar
+Import-Package: 
+ com.google.common.collect,
```

Comment:
We are just trying to phase out Guava - could you remove this import and change the one line where it is used to a pure Java8 construct?